app/assets/stylesheets/application.css.scss

```diff
@@ -6,3 +6,16 @@
 @import "daplog/base";
 @import "daplog/blocks";
 @import "daplog/contexts";
+
+#foo{}
+a.Bar   {
+    margin : 0 0 0;
+	backgrounds:url(https://example.com/assets/image.png) ;
+  & > { color: red; }
+  content: 'foo'! important;
+  -webkit-border-radius: 0px .500em;
+  border: none solid rgba(0,0,0,0); /* baz */
+  @extend #foo;
```

Comment:
Name of placeholder `#foo` should be written must match regex /^[a-zA-Z_][-a-zA-Z0-9_]*$/<br>Prefer using placeholder selectors (e.g. %some-placeholder) with @extend